src/mapping-v3-core.ts

```diff
@@ -432,6 +590,51 @@ function handleProjectSecondaryMarketRoyaltyPercentageUpdated(
   }
 }
 
+/**
+ * @notice Handle ProjectExternalAssetDependenciesLocked event. The event was
```

Comment:
I think the start of this comment is for a different handler?

Reply:
updated in [a475369](https://github.com/ArtBlocks/artblocks-subgraph/pull/289/commits/a47536973be141709ff17ddbbd662d5e335ffa26)